fat/fat2/transactionio.go

```diff
@@ -0,0 +1,67 @@
+package fat2
+
+import (
+	"fmt"
+	"github.com/Factom-Asset-Tokens/factom"
+)
+
+// TransactionIO represents an individual input or output of a fat2 transaction
```

Comment:
I'm not much of a fan of this type name. I find out that this is an input or output very clearly by looking at the Transaction type, so it unfortunately doesn't tell me much about what an input or output *is*. To me this looks more like a mapping between addresses and Amounts that are associated with a Type. This suggests that maybe another type should be used with this, a TypeAmount or Value type that associates addresses and types. Consider the type names for the fat0 and fat1 Transactions: AddressAmountMap, AddressNFTokensMap.

Reply:
Removed the whole struct in the refactor